nsis/SectionCoreReqs.nsh

```diff
@@ -53,20 +53,13 @@
     ; in order to run (including CDVD!) -- and really there should be more but we don't have working
     ; SPU2 null plugins right now.
  
-    File ..\bin\Plugins\GSnull.dll
+    ;File ..\bin\Plugins\GSnull.dll
```

Comment:
Honestly, I can't think of a use either. I'd still ship it though, as clearly someone thought it would be useful..